opossum-tool/OpossumUI
# Remove PackageSearchPopup

### Summary of changes

- Remove the PackageSearchPopup which is not needed anymore after the changes of #2410 and #2415
- Rename the SearchPackagesIcon to SearchSignalsIcon as it is now only used for signals.

### Context and reason for change

Closes #2397 

### How can the changes be tested

In Audit View create a new attribution and check there is no more search icon next to the package name field.
